docs/src/index.md

```diff
@@ -0,0 +1,68 @@
+# Pkg3.jl
+
+!!! warning
+    This documentation is a work in progress and the information in it might be or become outdated.
+
+Sections:
+
+```@contents
+Pages = [
+    "index.md"]
+```
+
+## Introduction
+
+Pkg3 is the package manager for Julia.
+
+## Getting Started
+
+The Pkg REPL-mode is entered using from the Julia REPL using the key `]`. To start a new project, use `pkg> new`.
+
+```
+pkg> new HelloWorld
+```
+
+This creates a new package `HelloWorld` with the following files
+
+```jl
+julia> cd("HelloWorld")
+shell> tree .
+.
+├── Project.toml
+└── src
+    └── HelloWorld.jl
```

Comment:
We should probably generate stubs for `test/runtests.jl` and `docs/index.md` as well since I find that the more you reduce the friction for these things the more likely people are to actually write them.

Reply:
Ok but then `Test` needs to go into the `project` and it needs a `compatability` and we need to discuss test-only dependencies etc.

Reply:
Fair enough, we can wait until that's sorted out.